plugins/smartopenselected.lua

```diff
@@ -0,0 +1,38 @@
+-- mod-version:3
+local core = require "core"
+local command = require "core.command"
+local keymap = require "core.keymap"
+local common = require "core.common"
+local contextmenu = require "plugins.contextmenu"
+
+
+command.add("core.docview!", {
```

Comment:
Might be a good idea to check if a selection exists in the command predicate.

Reply:
Default behavior is an error message in the status bar "no text selected", which seems appropriate.